airflow/providers/celery/executors/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
```

Comment:
Do we need to add minimum airflow version to all files under this folder?

We need to consider that users of lower Airflow versions will have newer Celery provider version and they might think that bug fixes made to the providers can be used with their executor

Reply:
Yes. Good idea. I think we have to raise an exception here if someone wants to use those executors.

Purely theorethically, those executors "COULD" work if user directly points to them even on earlier airflow versions, but I think both 2.5 and 2.6 had the "decoupling" changes implemented that likely make the executors "sort-of" working, so it's safer to check the airflow version and raise exception if those modules are used there. 

it will require few more modification (i.e. catching and ignoring the specific exception in "verify_providers" job, but should be possible in principle.

Reply:
Ah nice. We can use `AirflowOptionalProviderFeatureException` for that one and it will be automatically ignored by verify_providers, so it's even simpler than I thought.